index.html

```diff
@@ -2014,19 +1958,19 @@ <h3>Retrieve Verification Method</h3>
         <p>
 The following algorithm specifies how to safely retrieve a verification method,
 such as a cryptographic <a>public key</a>, by using a <a>verification method</a>
-identifier contained in a <a>data integrity proof</a>. Required inputs are a
-<strong><a>data integrity proof</a></strong> (<var>proof</var>) and a set of
-<strong>dereferencing options</strong> (<var>options</var>). A
+identifier. Required inputs are a
+<strong><a>verification method</a></strong> (<var>verificationMethod</var>),
+a <strong>proof purpose</strong> (<var>proofPurpose</var>), and
```

Comment:
`proofPurpose` isn't defined by this document, so I suggest removing this clause.